_posts/blog/2018-09-10/2018-09-10-the-beginners-guide-to-decentralization.markdown

```diff
@@ -0,0 +1,97 @@
+
+Centralization also has drawbacks. There may not be ambiguity about who is in control, but that doesn’t mean those in control are actually competent, or moral. Centralized systems are easier to build technically, but they are also more fragile, since they have a single point of failure (if the person / organization in charge is gone, so is the system). Being permissioned means that those in power might exclude people they don’t like or agree with.
+
+Centralized systems can change based on who is in control. Even if the system is run well by competent and moral people today, it could be run poorly by different people tomorrow.
+
+There is an alternative to having systems so tightly controlled: Decentralization.
+
+### What Are Decentralized Systems?
+
+As we’ve said, _decentralized_ means that there is no single person or organization that has control over everyone else in a given system.
+
+Participants in decentralized systems have the ability to make decisions themselves and aren’t forced to rely on anyone who controls the system. Let’s look at two examples.
+
+> **A decentralized political system** is one where no person or organization has the ability to govern everyone else. A completely decentralized political system would be an anarchy (absence of a governing structure). Modern democratic systems have some elements of decentralization, where participants in the system do have some control over the decision-making process and there are multiple organizations which exist to ensure power doesn’t become centralized within one portion of the government.
+
+> **A decentralized computer network** is a system where the computers talk directly to each other and there is no central point of control. Most online platforms today work by having a lot of computers called servers that their users are forced to connect to if they want to use the service. They are centralized and completely control the system. A decentralized computer network has no central servers that control access to the network; each users’ computer connects directly to other users’ computers, making a network of computers that no one controls. OpenBazaar is such a network, as are networks like Bitcoin and Bittorrent.
+
+This image shows what a decentralized system looks like. The participants in the system are equal and connect with each other instead of depending on a central party.
+
+![Visual model of a decentralized network](Decentralized-network.png "Visual model of a decentralized network")
+*Decentralized network with no center*
+
+Decentralized systems are _permissionless_. Since no one controls the system, there’s no one to stop people from joining or forcing them to leave. You can join or leave the OpenBazaar or Bitcoin networks anytime you want to.
```

Comment:
You're saying "there is no one to ... forcing them to leave".

I think you want something like "no one to [stop people from joining] or [force them to leave] (brackets to show the phrases).